lib/pre_assembly/remediation/remediate.rb

```diff
@@ -148,7 +148,7 @@ def log_info
       end
 
       def run_assembly_robot(name)
-        `BUNDLE_GEMFILE=~/assembly/current/Gemfile ROBOT_ENVIRONMENT=#{ENV['ROBOT_ENVIRONMENT']} bundle exec ~/assembly/current/bin/run_robot dor:assemblyWF:#{name} -e #{ENV['ROBOT_ENVIRONMENT']} -d #{@pid}`
+        `BUNDLE_GEMFILE=~/assembly/current/Gemfile RAILS_ENV=#{ENV['RAILS_ENV']} bundle exec ~/assembly/current/bin/run_robot dor:assemblyWF:#{name} -e #{ENV['RAILS_ENV']} -d #{@pid}`
```

Comment:
This will certainly break the system-level invocation of `assembly` code, since `assembly` has no knowledge of Rails.  So if you were going to keep one `ROBOT_ENVIRONMENT`, it would probably be this one.

Reply:
Thanks Joe, just updated and pushed changes.